src/engine/server.cpp

```diff
@@ -1934,6 +1934,7 @@ int main(int argc, char **argv)
     if(enet_initialize()<0) fatal("Unable to initialise network module");
     atexit(enet_deinitialize);
 
+#ifdef NDEBUG
```

Comment:
Please document why we decided to disable the signals for debug builds in a comment.